### PR TITLE
Verified checkmark

### LINKED ISSUE #148: Make it so paid members have a verified symbol next to their name
## Describe the feature
Make it so paid members have a verified symbol next to their name

### PR DESCRIPTION
**Pull request by HanielDiaz**
Fixes #148 

## Description
Added Verified checkmark to profile and leaderboard

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other (specify below)

## Checklist:

- [x] My code follows the code style of this project found on [Contribution guidelines](https://github.com/SHPEUCF/shpeucfapp/wiki/Contribution-guidelines).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/screens/General/Leaderboard.js [hunk 1]
```diff
@@ -78,24 +79,26 @@ class Leaderboard extends Component {
...
 						<Text style = { indexText }>{ item.index }</Text>
 					</View>
 					<View >
 						<View style = { userInfoContainer }>
 							<View style = { userTextContainer }>
-								<Text style = { [textStyle, { fontWeight: "bold" }] }>
-									{ `${item.firstName} ${item.lastName}` }
-								</Text>
-								<Text style = { [textStyle, { fontSize: dimension.width * 0.04 }] }>
+								<View style = { [row] }>
```
Comment (idelmr) [inline src/screens/General/Leaderboard.js:97]: Remove square brackets if only one style.

File: src/screens/General/Leaderboard.js [hunk 2]
```diff
@@ -78,24 +79,26 @@ class Leaderboard extends Component {
...
+									<Text style = { [textStyle, { fontWeight: "bold" }, currentUserTextStyle] }>{ `${item.firstName} ${item.lastName}` }</Text>
```
Comment (idelmr) [inline src/screens/General/Leaderboard.js:98]: Split `Text` in 3 lines.

File: src/utils/render.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import Ionicons from "react-native-vector-icons/Ionicons";
+
+// You pass in the privileges prop
+export const verifiedCheckMark = ({ paidMember }) => {
+	const {
+		verifiedCheckMark
+	} = styles;
```
Comment (idelmr) [inline src/utils/render.js:8]: Add newline after destructuring.

File: src/utils/render.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+	if (paidMember)
+		return (
+			<Ionicons
+
```
Comment (idelmr) [inline src/utils/render.js:12]: Remove empty line.

**Comment by esteban737** [comment]
I'm running the code

**Pull request merged**